### PR TITLE
 Add error logger for fulfillable request

### PR DESCRIPTION
**Pull request by indrekj**
OM-28

### CODE REVIEW

**Comment by SteamRiC** [comment]
+1

**Comment by SteamRiC** [comment]
Tests fail

**Comment by indrekj** [comment]
fixed

File: lib/salemove/process_handler/pivot_process.rb [hunk 1]
```diff
@@ -149,6 +149,7 @@ def handle_fulfillable_response(input, handler, response)
             end
           end
         rescue Timeout::Error
+          PivotProcess.logger.error "Fullfillable response was not fulfilled in #{timeout} seconds", input
```
Comment (willmore) [inline lib/salemove/process_handler/pivot_process.rb]: If logging this is a feature of the widely used library, it should be tested.

**Comment by SteamRiC** [comment]
Updated

**Comment by indrekj** [comment]
@willmore markus added a test

**Comment by willmore** [comment]
!squash

**Pull request merged**